woocommerce/woocommerce-ios
# Product creation AI: Present package flow

Closes: #10733 

## Description

Presents package photo flow from product creation AI flow.

## Testing instructions

- Create a JN site with Jetpack AI plugin. (Checkbox available in JN home page to install this plugin)
- Switch to Products tab, select the "+" button to add a new product. The same AI action sheet should be displayed.
- Tap on "Create a product with AI" to launch  "Add your product name" screen
- Tap on "Use package photo" button and validate that the package flow is launched.

## Screenshots
![Simulator Screen Recording - iPhone 14 Pro - 2023-09-19 at 16 19 27](https://github.com/woocommerce/woocommerce-ios/assets/524475/cfe4b750-b20a-4c71-a7ba-c0a6d8ba957e)

---
